cli/src/live-common-setup.ts

```diff
@@ -26,6 +26,7 @@ import { disconnectAll } from "@ledgerhq/live-common/lib/api";
 
 checkLibs({
   NotEnoughBalance,
+  // @ts-ignore
```

Comment:
If this is because you can't get the cli to build on windows, add a comment on it so we know why

Reply:
ok done